include/TPP/Passes.td

```diff
@@ -25,6 +25,7 @@ def LoadTppDialects : Pass<"load-tpp-dialects", "ModuleOp"> {
                            "check::CheckDialect",
                            "perf::PerfDialect",
                            "omp::OpenMPDialect",
+                           "amx::AMXDialect",
```

Comment:
Not really a TPP dialect, how come it's not pulled in with all the other upstream ones?